java/src/jmri/jmrix/can/cbus/CbusCabSignalManager.java

```diff
@@ -0,0 +1,49 @@
+package jmri.jmrix.can.cbus;
+
+import jmri.CabSignal;
+import jmri.implementation.DefaultCabSignal;
+import jmri.jmrix.can.CanSystemConnectionMemo;
+import jmri.LocoAddress;
+import jmri.managers.AbstractCabSignalManager;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * CBUS implementation of the {@link jmri.CabSignalManager} interface.
+ *
+ * <hr>
+ * This file is part of JMRI.
+ * <P>
+ * JMRI is free software; you can redistribute it and/or modify it under the
+ * terms of version 2 of the GNU General Public License as published by the Free
+ * Software Foundation. See the "COPYING" file for a copy of this license.
+ * <P>
+ * JMRI is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+ * A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ * <P>
+ *
+ * @author Paul Bender Copyright (C) 2019
+ * @author Steve Young Copyright (C) 2019 
+ */
+public class CbusCabSignalManager extends AbstractCabSignalManager {
+
+    private CanSystemConnectionMemo _memo = null;
+
+    public CbusCabSignalManager(CanSystemConnectionMemo memo){
+        super();
+        _memo = memo;
+        log.debug("LocoNet Cab Signal Manager initialized");
```

Comment:
You probably want to change this debug message to refer to the right system.

Reply:
Good catch :-)